File: data/src/main/java/com/platdmit/data/api/models/ApiAuth.java
package com.platdmit.data.api.models;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class ApiAuth {
    @SerializedName("account")
    @Expose
    private ApiRequestBody account;

    @SerializedName("session_key")
    @Expose
    private String sessionKey;

    public ApiRequestBody getAccount() {
        return account;
    }

    public void setAccount(ApiRequestBody account) {
        this.account = account;
    }

    public String getSessionKey() {
        return sessionKey;
    }

    public void setSessionKey(String sessionKey) {
        this.sessionKey = sessionKey;
    }
}

File: mod_login/src/main/java/com/platdmit/mod_login/data/retrofit/models/ApiAuth.kt
package com.platdmit.mod_login.data.retrofit.models

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

data class ApiAuth (
        @SerializedName("account")
        @Expose
        val requestBody: ApiRequestBody,
        @SerializedName("session_key")
        @Expose
        val sessionKey : String
)